pkg/server/plugin/nodeattestor/aws/iid.go

```diff
@@ -73,17 +75,21 @@ type IIDAttestorPlugin struct {
 		// in test, this can be overridden to mock OS env
 		getEnv func(string) string
 	}
+	log hclog.Logger
 }
 
 // IIDAttestorConfig holds hcl configuration for IID attestor plugin
 type IIDAttestorConfig struct {
-	caws.SessionConfig   `hcl:",squash"`
-	SkipBlockDevice      bool   `hcl:"skip_block_device"`
-	SkipEC2AttestCalling bool   `hcl:"skip_ec2_attest_calling"`
-	AgentPathTemplate    string `hcl:"agent_path_template"`
-	pathTemplate         *template.Template
-	trustDomain          string
-	awsCaCertPublicKey   *rsa.PublicKey
+	caws.SessionConfig    `hcl:",squash"`
+	SkipBlockDevice       bool     `hcl:"skip_block_device"`
+	LocallyTrustedAcctIDs []string `hcl:"locally_trusted_account_ids"`
```

Comment:
Hmm I'm still not sure that this accurately conveys the purpose and functionality of the configurable. I only harp on this because these things are hard to change once they go in. I'd really prefer something that communicates the fact that 1) this is a list of account IDs, 2) we will validate the IID locally i.e. we are still checking its signature, and 3) we will not call out to AWS APIs

I think this name misses on point 2. How about `account_ids_for_local_validation`?

I'd like to get this merged in today, curious to hear naming thoughts or last requests on this pr from @azdagron

Reply:
I agree with the naming concern.

Reply:
The rest of the PR looks good.